### PR TITLE
Added XE ISO2Box create script

### PR DESCRIPTION
**Pull request by rschmied**
**Added** 
- `include/embedded_vagrantfile_xe`
- `iosxe_iso2vbox.py`
- `README-CSR1000v.md`

**Changed**
- `.gitignore` (to supress virtualenv directories and test artifacts)

**Comment**
I did not change iosxr_iso2vbox.py even though it shows in the diff. I did a local directory compare and the changes shown seem to be from the latest commits to ios-xr/iosxrv-x64-vbox. I did a pull from the repo before applying my changes and it shows clean on my end. Something went wrong somewhere but I can't tell why / where (I **am** a Git noob at the end).

### CODE REVIEW

File: iosxe_iso2vbox.py [hunk 1]
```diff
@@ -0,0 +1,626 @@
...
+    parser.add_argument('-o', '--create_ova', action='store_true',
+                        help='additionally use VBoxManage to export an OVA')
+    parser.add_argument('-d', '--debug', action='store_true',
+                        help='will exit with the VM in a running state. Use: socat TCP:localhost:65000 -,raw,echo=0,escape=0x1d to access')
+    parser.add_argument('-n', '--nocolor', action='store_true',
+                        help='don\'t use colors for logging')
+    parser.add_argument('-v', '--verbose',
+                        action='store_const', const=logging.INFO,
+                        default=logging.WARN, help='turn on verbose messages')
+    args = parser.parse_args()
```
Comment (RichWellum) [inline iosxe_iso2vbox.py:383]: Maybe consider adding the Unit-Test capabilities at some point in the future.

File: iosxe_iso2vbox.py [hunk 2]
```diff
@@ -0,0 +1,626 @@
...
+    logger.critical("Use: 'socat TCP:localhost:65000 -,raw,echo=0,escape=0x1d' to access the VM")
+    raw_input("Press Enter to continue.")
+    # To debug post box creation, add the following line to Vagrantfile
+    # config.vm.provider "virtualbox" do |v|
+    #   v.customize ["modifyvm", :id, "--uart1", "0x3F8", 4, "--uartmode1", 'tcpserver', 65000]
+    # end
+
+
+def start_process(args):
+    """ Start vboxheadless process
```
Comment (RichWellum) [inline iosxe_iso2vbox.py]: Personal preference is to do:
"""
comment here
"""

But not a big issue

**Review by RichWellum — COMMENTED** [review]
Looks good Ralphs - couple of small comments.

Assuming you ran flake8 on the code?

Also before you merge I want to understand the changes to my script - will talk it over with our resident git master Glenn.

**Comment by rschmied** [comment]
As stated, the XR script change should simply be discarded from the PR. I did a PEP8 autoformat of the code from within Sublime but can install and run Flake8.

**Comment by glennmatthews** [comment]
Merged manually to fix the XR oddness. Thanks for the contribution!

**Pull request closed without merging**